CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ FEATURES:
 ENHANCEMENTS:
 * Add cran support to nexus, open port 80 for the workspace nsg and update the firewall config to allow let's encrypt CRLs ([#2694](https://github.com/microsoft/AzureTRE/pull/2694))
 * Upgrade Github Actions versions ([#2731](https://github.com/microsoft/AzureTRE/pull/2744))
+* Install TRE CLI inside the devcontainer image (rather than via a post-create step) ([#TBD](https://github.com/microsoft/AzureTRE/pull/TBD))
```

Comment:
```suggestion
* Install TRE CLI inside the devcontainer image (rather than via a post-create step) ([#2757](https://github.com/microsoft/AzureTRE/pull/TBD))
```